src/local_scheduler/local_scheduler_client.h

```diff
@@ -82,7 +82,9 @@ void local_scheduler_log_event(LocalSchedulerConnection *conn,
                                uint8_t *key,
                                int64_t key_length,
                                uint8_t *value,
-                               int64_t value_length);
+                               int64_t value_length,
+                               double time
```

Comment:
- Can we use `timestamp` instead of `time`? 
- Move the parenthesis onto the line before, so it is `double timestamp);`
- Change the documentation above the function to include the new argument.